File: app/src/main/java/com/app/rupiksha/models/WalletReportModel.java
package com.app.rupiksha.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WalletReportModel
{
    @SerializedName("id")
    @Expose
    private int id;
    @SerializedName("uid")
    @Expose
    private String uid;
    @SerializedName("txn_uid")
    @Expose
    private Object txnUid;
    @SerializedName("type")
    @Expose
    private String type;
    @SerializedName("txntype")
    @Expose
    private String txntype;
    @SerializedName("remark")
    @Expose
    private String remark;
    @SerializedName("txnid")
    @Expose
    private String txnid;
    @SerializedName("amount")
    @Expose
    private String amount;
    @SerializedName("opening")
    @Expose
    private String opening;
    @SerializedName("closing")
    @Expose
    private String closing;
    @SerializedName("date")
    @Expose
    private String date;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getUid() {
        return uid;
    }

    public void setUid(String uid) {
        this.uid = uid;
    }

    public Object getTxnUid() {
        return txnUid;
    }

    public void setTxnUid(Object txnUid) {
        this.txnUid = txnUid;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getTxntype() {
        return txntype;
    }

    public void setTxntype(String txntype) {
        this.txntype = txntype;
    }

    public String getRemark() {
        return remark;
    }

    public void setRemark(String remark) {
        this.remark = remark;
    }

    public String getTxnid() {
        return txnid;
    }

    public void setTxnid(String txnid) {
        this.txnid = txnid;
    }

    public String getAmount() {
        return amount;
    }

    public void setAmount(String amount) {
        this.amount = amount;
    }

    public String getOpening() {
        return opening;
    }

    public void setOpening(String opening) {
        this.opening = opening;
    }

    public String getClosing() {
        return closing;
    }

    public void setClosing(String closing) {
        this.closing = closing;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }
}

File: app/src/main/java/com/app/rupiksha/models/WalletReportModel.kt
package com.app.rupiksha.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class WalletReportModel(
    @SerializedName("id")
    @Expose
    val id: Int,

    @SerializedName("uid")
    @Expose
    val uid: String? = null,

    @SerializedName("txn_uid")
    @Expose
    val txnUid: Any? = null,

    @SerializedName("type")
    @Expose
    val type: String? = null,

    @SerializedName("txntype")
    @Expose
    val txntype: String? = null,

    @SerializedName("remark")
    @Expose
    val remark: String? = null,

    @SerializedName("txnid")
    @Expose
    val txnid: String? = null,

    @SerializedName("amount")
    @Expose
    val amount: String? = null,

    @SerializedName("opening")
    @Expose
    val opening: String? = null,

    @SerializedName("closing")
    @Expose
    val closing: String? = null,

    @SerializedName("date")
    @Expose
    val date: String? = null
)
